lib/config.ml

```diff
@@ -39,8 +39,6 @@ module Ciphers = struct
     `TLS_RSA_WITH_AES_256_CBC_SHA ;
     `TLS_RSA_WITH_AES_128_CBC_SHA ;
     `TLS_RSA_WITH_3DES_EDE_CBC_SHA ;
-    `TLS_RSA_WITH_RC4_128_SHA ;
-    `TLS_RSA_WITH_RC4_128_MD5
     ]
```

Comment:
Well, this is a bit of a lie; removing ciphers we support from the list of supported ciphers is, um... yeah.

Nothing that couldn't be helped by a little rename and little rethink of what this list exactly means, or what potential similar lists defined next to it could mean.